test/docker_tests/test_ldap_ui.py

```
@@ -0,0 +1,198 @@
+        container_name = f"{project_name}-netalertx-1"
+        
+        # Give PHP/Nginx extra time to fully bootstrap UI
+        time.sleep(10)
+        
+        print(f"Running UI test inside {container_name}...")
+        cmd = [
+            "docker", "exec", container_name,
+            "pytest", "/app/test/ui/test_ui_ldap_login.py", "-v"
+        ]
+        
+        result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)